build/finish-release.ps1

```diff
@@ -0,0 +1,67 @@
+Param(
+    [Parameter(Mandatory=$true)]
+    [string] $version,
+    [ValidateSet('release', 'hotfix')]
+    [Parameter(Mandatory=$true)] [string] $release_type
+)
+
+$fhir_releases = @('stu3', 'r4', 'r5')
+
+Push-Location $PsScriptRoot\..
+
+function TagAndPush([string] $release_branch, [string] $tag_name)
+{
+    Write-Host "Tag $release_type with $tag_name"
+
+    # go to release branch
+    git checkout $release_branch
+
+    # tag the release
+    git tag -a -m "version $tag_name" $tag_name
+
+    # push commits and tags
+    git push --follow-tags --set-upstream origin $release_branch
+}
+
+function Merge-Push([string] $source_branch, [string] $target_branch)
+{
+    Write-Host "Merge and push $source_branch into $target_branch" 
+
+    # save current branch name
+    $current = git branch --show-current
+
+    # go to target branch
+    git checkout $target_branch
+
+    # merge source branch
+    git merge --no-ff origin/$source_branch
```

Comment:
Why `--no-ff`?

Reply:
Indeed `--no-ff` (No fast forward), because this is meant to merge the release branch to master or merge the release branch back to develop again.  Manually I always created a merge commit (instead of fast forward merge). 
Fast forward would work for master I think. I am not sure it would work for develop...